sponsor/index.html

```diff
@@ -123,75 +132,242 @@ <h4>Sponsorship via Invoice</h4>
+           <p>
+           Please see attached logo for OSGeo Bronze sponsorship level.</p>
+      
+           <p>Anaximander<br/>
+      
+           <p>attachement: <i>milesian.svg</i></p>
+         </blockquote>
       </li>
     </ol>
     
-    <h4>Donate via GitHub Sponsors</h4>
+    <h4>Donate via PayPal</h4>
     <p>
-       Great for monthly donations, provides some sponsorship matching, a github sticker, and avoid PayPal overhead:
+       Easy for one-time donations and sponsorships (although PayPal takes a small overhead):
     </p>
     <ol>
-      <li>Visit GitHub Sponsors page for <a href="https://github.com/sponsors/OSGeo">https://github.com/sponsors/OSGeo</a>
+      <li>OSGeo provided a PayPal donate button that goes straight to GeoServer:<br/>
+        <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=4XZ76BZMEWZ24&source=url">
+          <img src="../img/btn_donateCC_LG.gif" width="147" height="47"/>
+        </a>
+      </li>
+      <li>Check the currency when donating (USD rates are provided by default). <br/>
+          <img src="img/paypal-donation.png" width="417"/>
       </li>
-      <li>Select a monthly donation sponsorship tier.
+      <li>Use the message field to support a specific activity: <br/>
+          <img src="img/paypal-message.png" width="417"/>
       </li>
       <li>
-         The OSGeo Treasurer will contact both you and the GeoServer PSC to acknowledge your sponsorship:
+         The OSGeo Treasurer will contact both you and the GeoServer PSC to acknowledge your sponsorship.
          <ul>
-         <li>Please have a logo ready for your organization if you wish to be recognized publicly.</li>
-         <li>As with any OSGeo activity your sponsorship (at your request) will be cross listed on the OSGeo sponsorship page.</li>
+         <li>Donations: We will ask individuals for their name, and a useful link.</li>
+         <li>Sponsorship: We will ask organisations for their logo to be listed on the GeoServer and OSGeo website.</li>
          </ul>
       </li>
-      <li>
-        Example email requesting listing on the OSGeo sponsorship page.
-        <blockquote>
-          <p>Subject: setting up geoserver sponsorship</p>
-          <p>To: treasurer@osgeo.org</p>
-
-          <p>Treasurer:</p>
-          <p>
-          I have used GitHub Sponsors page to setup a monthly donation of $50 to the GeoServer project.</p>
-          <p>
-          Please observe the attached logo for OSGeo Bronze sponsorship.</p>
-     
-          <p>Anaximander<br/>
-     
-          <p>attachement: <i>milesian.svg</i></p>
-        </blockquote>
-      </li>
     </ol>
+    
 </section>
+
+<section id="recommendations">
+    <h2>Sponsorship Recognition and Recommendations</h2>
+    <p class="lead">
+      Financial support indicates a commitment to the sustainability of the GeoServer, for those that depend on GeoServer's availability. Sponsorship is not a replacement for commercial support.
```

Comment:
```suggestion
      Financial support indicates a commitment to the sustainability of the GeoServer, for those that depend on GeoServer's availability.
```

"Sponsorship is not a replacement for commercial support" needs its own place IMHO, explaining that it's not directed to a specific goal, unlike commercial support

Reply:
Except sometimes we do fundraising for a specific goal; the point is they do not get to choose the goal :)

If I start the sentence with that statement would it be visible enough?

Reply:
Yes